src/puppetlabs/dujour/version_check.clj

```diff
@@ -118,7 +128,21 @@
   ([request-values update-server-url callback-fn]
     (validate-config! request-values update-server-url)
     (future
-      (let [server-response (try
+      (let [certname (:certname request-values)
+            request-values (if-not (nil? certname)
+                             (->
+                               request-values
+                               (assoc :host-id (get-hash certname))
+                               (dissoc :certname))
+                             request-values)
+            cacert (:cacert request-values)
+            request-values (if-not (nil? cacert)
+                             (->
+                               request-values
+                               (assoc :site-id (get-hash cacert))
+                               (dissoc :cacert))
+                             request-values)
```

Comment:
I think it's better to keep them as `certname` and `cacert` for the request to this library. That way, there's no misunderstanding about what value is being sent in. Logically, we can consider the `site-id` and `host-id` values to be what occurs after the hashing operation.